tensorflow_addons/utils/keras_utils.py

```diff
@@ -21,6 +21,7 @@
 
 # TODO: find public API alternative to these
 from tensorflow.python.keras.losses import LossFunctionWrapper  # pylint: disable=unused-import
+from tensorflow.python.keras.metrics import Metric  # pylint: disable=unused-import
```

Comment:
What was the issue with using the public `tf.keras.metrics.Metric`?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/keras/metrics.py#L65
```
>>> import tensorflow as tf
>>> tf.keras.metrics.Metric
<class 'tensorflow.python.keras.metrics.Metric'>
```

Reply:
Sean, I believe you're right.

Reply:
```suggestion
```

Reply:
I removed this